Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "asdf-core-rust"
+version = "0.1.0"
+edition = "2021"
+
+description = "Extendable version manager with support for Ruby, Node.js, Elixir, Erlang & more - now in Rust!"
```

Comment:
Copied from the Github interface (with an addition). Not sure this will actually get displayed anywhere.